doc/release-notes.md

```diff
@@ -28,207 +26,77 @@ likely require a reindex.
-
-- **LLMQ Type Enhancement**
-    - **LLMQ_50_60** is enabled for **Devnet** networks.
-    - Necessary for testing on a large Devnet.
-
-## Tests
-
-- **Regtest Network Softfork Activation Heights**
-    - For the `regtest` network, the activation heights of several softforks have been set to **block height 1**.
-    - **Customization:** Can be changed using the runtime setting `-testactivationheight=name@height`.
-    - *(dash#6214)*
-
-## Statistics
-
-### New Features
-
-- **Statsd Client Enhancements**
-    - Supports queueing and batching messages.
-        - **Benefits:**
-            - Reduces the number of packets.
-            - Lowers the rate at which messages are sent to the Statsd daemon.
-
-- **Batch Configuration Options**
-    - **Maximum Batch Size:**
-        - Adjustable using `-statsbatchsize` (in bytes).
-        - **Default:** **1KiB**.
-    - **Batch Send Frequency:**
-        - Adjustable using `-statsduration` (in milliseconds).
-        - **Default:** **1 second**.
-        - **Note:** `-statsduration` does not affect `-statsperiod`, which dictates how frequently some stats are *collected*.
-
-### Deprecations
-
-- **Deprecation of `-platform-user`**
-    - `-platform-user` is deprecated in favor of the whitelist feature.
-    - In releases **22.x** of Dash Core, it has been renamed to `-deprecated-platform-user`.
-    - It will be removed in version **23.x**.
+## New RPCs
 
-- **`-statsenabled` Deprecation**
-    - Now implied by the presence of `-statshost`.
-    - It will be removed in version **23.x**.
+- **`getislocks`**
+    - Retrieves the InstantSend lock data for the given transaction IDs (txids).
+      Returns the lock information in both human-friendly JSON format and binary hex-encoded zmq-compatible format.
 
-- **`-statshostname` Replacement**
-    - Deprecated and replaced with `-statssuffix` for better representation of the argument's purpose.
-    - **Behavior:** Behave identically to each other.
-    - It will be removed in version **23.x**.
+Updated RPCs
+------------
 
-- **`-statsns` Replacement**
-    - Deprecated and replaced with `-statsprefix` for better representation of the argument's purpose.
-    - **Behavior:** `-statsprefix` enforces the usage of a delimiter between the prefix and key.
-    - It will be removed in version **23.x**.
+- The top-level fee fields `fee`, `modifiedfee`, `ancestorfees` and `descendantfees`
+  returned by RPCs `getmempoolentry`,`getrawmempool(verbose=true)`,
+  `getmempoolancestors(verbose=true)` and `getmempooldescendants(verbose=true)`
+  are deprecated and will be removed in the next major version (use
+  `-deprecated=fees` if needed in this version). The same fee fields can be accessed
+  through the `fees` object in the result. WARNING: deprecated
+  fields `ancestorfees` and `descendantfees` are denominated in sats, whereas all
+  fields in the `fees` object are denominated in DASH.
```

Comment:
I was looking at this today and per the docs these have been deprecated since 0.17.0. Are we sure something changed?

Reply:
@kwvg clarified for me. You can ignore the original comment and mark it resolved.